les/fetcher.go

```diff
@@ -576,13 +577,19 @@ func (f *lightFetcher) processResponse(req fetchRequest, resp fetchResponse) boo
 		headers[int(req.amount)-1-i] = header
 	}
 
-	if _, err := f.chain.InsertHeaderChain(headers, 1); err != nil {
+	_, err := f.chain.InsertHeaderChain(headers, 1)
```

Comment:
?

Reply:
just ignore it. :P

The entire les.fetcher is already refactored. I just need to rebase.